Fix/sync app and ros param

PerrineAguiar opened the pull request:
Fixes #224.
Fixes #221.

It also fixes some errors regarding the GetMapUuids service.

smits commented on tango_ros_common/tango_ros_common/src/main/java/eu/intermodalics/tango_ros_common/TangoServiceClientNode.java:
```diff
@@ -171,7 +171,7 @@ private Boolean getMapUuids() throws ServiceNotFoundException {
                 mConnectedNode.newServiceClient(
                         NodeNamespaceHelper.BuildTangoRosNodeNamespaceName(GET_MAP_UUIDS_SRV_NAME),
-                        SaveMap._TYPE);
+                        GetMapUuids._TYPE);
```
There seems to be mismatch here between saveMapService and GetMapUuids._TYPE?  Shouldn't `saveMapService` be renamed to `getMapUuidsService`?

PerrineAguiar commented:
@smits Thanks for your review. I addressed your comments.
Here is the current behavior:
The preferences are set from the ROS params when:
* starting the app
* connecting to Tango
The ROS params are set from the app preferences when:
* pressing the RESTART TANGO button on the snackbar.

mcopejans commented on tango_ros_common/tango_ros_native/src/tango_ros_node.cpp:
```diff
@@ -35,15 +37,11 @@ std::vector<std::string> splitCommaSeparatedString(const std::string& comma_sepa
+  while (std::getline(ss, string_element, ',')) {
+    output.push_back(string_element);
+  }
+  return output;
```
Sorry for making the whitespace error. FYI: Intermodalics style guide is 4 spaces :)

smits replied:
Actually it is 2 if I'm not mistaken, except for older projects that still use 4.

mcopejans replied:
Oops, I still had an old code style file... Sorry :)

mcopejans commented on TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/ParameterNode.java:
```diff
@@ -81,4 +80,24 @@ public void uploadPreferencesToParameterServer() {
+                editor.putBoolean(paramName, booleanValue);
+            }
+            if (mParamNames.get(paramName) == "int_as_string") {
+                Integer intValue = mConnectedNode.getParameterTree().getInteger(NodeNamespaceHelper.BuildTangoRosNodeNamespaceName(paramName), 1);
```
Is there a reason for choosing 1 as default value instead of zero?

PerrineAguiar commented:
@mcopejans Thanks for your review.
